src/app/app/app-providers/overlay-source-providers/idaho-source-provider.ts

```diff
@@ -43,15 +46,20 @@ export class IdahoSourceProvider extends BaseOverlaySourceProvider {
 			.catch(this.handleError);
 	};
 
-	public fetch(fetchParams: IFetchParams): Observable<Overlay[]> {
+	public fetch(fetchParams: IFetchParams): Observable<OverlaysFetchData> {
 		// Multiple Source Provider may send a MultiPolygon which Idaho can't handle
 		if (fetchParams.region.type === 'MultiPolygon') {
 			fetchParams.region = geojsonMultiPolygonToPolygon(fetchParams.region as GeoJSON.MultiPolygon);
 		}
 
 		let url = this._overlaySourceConfig.baseUrl.concat(this._overlaySourceConfig.overlaysByTimeAndPolygon);
-		return <Observable<Overlay[]>>this.http.post(url, fetchParams)
+
+		// add 1 to limit - so we'll know if provider have more then X overlays
+		const requestParams  = Object.assign({}, fetchParams, {limit: fetchParams.limit + 1});
```

Comment:
please support no config value - meaning, make this line under an if (`if (fetchparams.limit)`)

Reply:
Will do